src/ng/sanitizeUri.js

```diff
@@ -44,7 +46,8 @@ function $$SanitizeUriProvider() {
    * Any url about to be assigned to img[src] via data-binding is first normalized and turned into
    * an absolute url. Afterwards, the url is matched against the `imgSrcSanitizationWhitelist`
    * regular expression. If a match is found, the original url is written into the dom. Otherwise,
-   * the absolute url is prefixed with `'unsafe:'` string and only then is it written into the DOM.
+   * the absolute url is prefixed with `'unsafe:'` string and only then is it written into the DOM,
+   * making it inactive.
    *
```

Comment:
Below in this file, I would replace the isImage argument with isMediaUrl to be clearer. That shouldn't break anything.